ports/cavecrawler2/port.json

```
@@ -0,0 +1,39 @@
+{
+  "version": 4,
+  "name": "cavecrawler2.zip",
+  "items": [
+    "Cave Crawler 2.sh",
+    "cavecrawler2"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Cave Crawler 2",
+    "porter": [
+      "NotYourAveragePaladin"
+    ],
+    "desc": "Cave Crawler 2 is a retro-inspired 2D side-scroller. Escape a series of winding caves full of strange creatures and treacherous platforming challenges.",
+    "desc_md": null,
+    "inst": "Ready To Run!!!",
+    "inst_md": null,
+    "genres": [
+      "platformer"
+    ],
+    "image": null,
+    "rtr": true,
+    "exp": false,
+    "runtime": [
+      "frt_4.3.squashfs",
```

```suggestion
      "godot_4.3.squashfs",
```
There's no FRT runtime for 4.3, you need the Godot 4.3 runtime.